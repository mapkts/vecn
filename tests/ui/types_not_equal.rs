#[vecn::vector]
struct TupleStruct(f32, f64);

#[vecn::vector]
struct NamedStruct {
    x: f32,
    y: f64,
}

fn main() {}
