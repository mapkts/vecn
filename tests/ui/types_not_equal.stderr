error: all fields of this struct should be of the same type
 --> $DIR/types_not_equal.rs:2:19
  |
2 | struct TupleStruct(f32, f64);
  |                   ^^^^^^^^^^

error: all fields of this struct should be of the same type
 --> $DIR/types_not_equal.rs:5:20
  |
5 |   struct NamedStruct {
  |  ____________________^
6 | |     x: f32,
7 | |     y: f64,
8 | | }
  | |_^
